pkg/ffmpeg/media_detection.go

```diff
@@ -0,0 +1,66 @@
+package ffmpeg
+
+import (
+	"bytes"
+	"github.com/stashapp/stash/pkg/logger"
+	"os"
+)
+
+// detect file format from magic file number
+// https://github.com/lex-r/filetype/blob/73c10ad714e3b8ecf5cd1564c882ed6d440d5c2d/matchers/video.go
+
+func mkv(buf []byte) bool {
+	return len(buf) > 3 &&
+		buf[0] == 0x1A && buf[1] == 0x45 &&
+		buf[2] == 0xDF && buf[3] == 0xA3 &&
+		containsMatroskaSignature(buf, []byte{'m', 'a', 't', 'r', 'o', 's', 'k', 'a'})
```

Comment:
It should be possible to construct the byte slice from a string literal rather than an array of bytes.

Reply:
The specific code was pullled as is from the project mentioned. Although it should work if we change it as you say, i think it's in this format to emphasize that  it is being used for binary reading a file and to make sure the exact byte count is correct. 
This probably explains it better https://medium.com/@xoen/go-strings-vs-byte-slices-5beb4c411bda.
If it's for cosmetic reasons i would like to keep it as is.

Reply:
Given that you've documented the origin, I'm cool with this.